README.rst

```diff
@@ -118,6 +239,168 @@ To stop fiware-cloto, execute:
 
     service fiware-cloto stop
```

Comment:
As said before, use ``::`` and prepend the prompt (and maybe "sudo"): ``$ sudo service fiware-cloto stop``